Migration Charts for Settings from iAPS to Trio

tmhastings opened the pull request:
Creates a page dedicated to assisting with locating and identifying settings entered during onboarding when a user is migrating from iAPS as requested by HCPs.

Also adds files to create similar charts for Loop, Trio 0.2, and AAPS.

tmhastings committed:
Migration Charts for Settings from iAPS to Trio

Creates a page dedicated to assisting with locating and identifying settings entered during onboarding when a user is migrating from iAPS as requested by HCPs.

Also adds files to create similar charts for Loop, Trio 0.2, and AAPS.

marionbarker commented on docs/configuration/migration/iaps-migration.md:
```diff
@@ -0,0 +1,71 @@
+
+!!! tip
+    To convert a value from a decimal to a percentage, multiply it by 100.  
+    To conver from a percentage to a decimal, divide by 100.
```
convert, not conver

marionbarker commented on docs/configuration/migration/iaps-migration.md:
```diff
@@ -0,0 +1,71 @@
+| Trio Name | Setting Format <br> (example) | Location in App | iAPS Name | Setting Format <br> (example) | Location in App |
+|:---:|:---:|:---:|:---:|:---:|:---:|
+| **[Glucose Targets](../settings/therapy/glucose-targets.md#how-to-enter-your-glucose-targets-into-trio)** | decimal <br> (100 mg/dL &#124; 5.5 mmol/L) | Settings → Therapy → Glucose Targets | **Target Glucose** | decimal <br> (100 mg/dL &#124; 5.5 mmol/L) | Settings → Target Glucose |
+| **[Basal Rates](../settings/therapy/basal-rates.md#how-to-enter-your-basal-profiles-into-trio)** | decimal <br> (1.0 U/hr) | Settings → Therapy → Basal Rates | **Basal Profile** | decimal <br> (1.0 U/hr) | Settings → Basal Profile | 
```
Use actual default values for Trio and iAPS.
Here are the default values for Trio that don't match what is in the table
* glucose target: 110 mg/dL, 6.1 mmol/L
* basal rate is 0.1 U/hr
* CR 30 g/U
* ISF 200 mg/dL/U, 11.1

Here are the default values for iAPS that don't match what's in the table
* glucose target: 99 mg/dL, 5.5 mmol/L

tmhastings replied:
I wasn't using the default values necessarily. Just giving examples, and some just happened to coincide with defaults. 
I also purposefully made them the same value so users can see what the equivalent would be in each app.

marionbarker commented on docs/configuration/migration/iaps-migration.md:
```diff
@@ -0,0 +1,71 @@
+| [**Enable UAM**](../settings/algorithm/smb-settings.md#enable-uam) | toggle <br> (On/Off) | Settings → Algorithm → Super Micro Bolus (SMB) → Enable UAM | **Enable UAM** | toggle <br> (On/Off) | Settings → OpenAPS → Enable UAM |
+| [**Max SMB Basal Minutes**](settings/algorithm/smb-settings.md#max-smb-basal-minutes) | decimal <br> (30 min) | Settings → Algorithm → Super Micro Bolus (SMB) → Max SMB Basal Minutes | **Enable UAM** | decimal <br> (30) | Settings → OpenAPS → Max SMB Basal Minutes |
+| [**Max UAM Basal Minutes**](settings/algorithm/smb-settings.md#max-uam-basal-minutes) | decimal <br> (30 min) | Settings → Algorithm → Super Micro Bolus (SMB) → Max UAM Basal Minutes | **Max UAM SMB Basal Minutes** | decimal <br> (30) | Settings → OpenAPS → Max UAM SMB Basal Minutes |
+| [**Max Delta-BG Threshold SMB**](settings/algorithm/smb-settings.md#max-delta-bg-threshold-smb) | percentage <br> (20%) | Settings → Algorithm → Super Micro Bolus (SMB) → Max Delta-BG Threshold SMB | **Max Delta-BG Threshold SMB** | decimal <br> (0.2) | Settings → OpenAPS → Max Delta-BG Threshold SMB |
```
I think for Trio, this is called Max Allowed Glucose Rise for SMB, Trio default is 20%

marionbarker commented on docs/usage/index.md:
```diff
@@ -16,17 +16,26 @@ Please use the navigation menu :fontawesome-solid-bars:, search bar :fontawesome
     A detailed look at Trio's user interface and features
 
-<!-- -   __[Temporary Targets ✏️](usage/temptarget.md)__
+-   __[Temporary Targets ✏️](usage/temptarget.md)__
```
remove "usage/". These files are in the same folder: should be just temptarget.md, same for overrides.md and statistics.md.

tmhastings replied:
Thanks! I also caught some that were missing the `.../` before the folder as well.

marionbarker commented on docs/configuration/migration/iaps-migration.md:
```diff
@@ -0,0 +1,71 @@
+| **[Max Bolus](../settings/therapy/units-limits.md#max-bolus)** | decimal <br> (10 U) | Settings → Therapy → Units and Limits → Max Bolus | **Max Bolus** | decimal <br> (10) | Settings → Pump Settings → Max Bolus |
+| **[Max Basal Rate](../settings/therapy/units-limits.md#max-basal)** | decimal <br> (2 U/hr) | Settings → Therapy → Units and Limits → Max Basal Rate | **Max Basal** | decimal <br> (2) | Settings → Pump Settings → Max Bolus |
+| **[Max COB](../settings/therapy/units-limits.md#max-cob)** | decimal <br> (120 g) | Settings → Therapy → Units and Limits → Max COB | **Max COB** | decimal <br> (120) | Settings → OpenAPS → Max COB |
+| **[Minimum Safety Threshold](../settings/therapy/units-limits.md#minimum-safety-threshold)** | decimal <br> (60 mg/dL) | Settings → Therapy → Units and Limits → Max IOB | **Threshold Setting** | decimal <br> (60) | Settings → Dynamic ISF → Threshold Setting |
```
Found one more thing for the table: Minimum Safety Threshold 
for Trio, 60 mg/dL, 3.3 mmol/L
for iAPS, 65 mg/dL, 3.6 mmol/L

But since this is set by the algorithm, with a target glucose of 110, the algorithm modifies this to 75 mg/dL.

tmhastings replied:
See [comment](https://github.com/nightscout/trio-docs/pull/114#discussion_r2083616111) regarding the values used.

tmhastings committed:
fix(configuration & usage): broken links & setting name

- Fix broken links and typos
- Update name of Max Delta BG Threshold to new name Max Allowed Glucose Rise for SMB
